system_tests/timeboost_test.go

```diff
@@ -41,6 +43,90 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestTimeboosBulkBlockMetadataAPI(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	builder := NewNodeBuilder(ctx).DefaultConfig(t, false)
+	cleanup := builder.Build(t)
+	defer cleanup()
+
+	arbDb := builder.L2.ConsensusNode.ArbDB
+	dbKey := func(prefix []byte, pos uint64) []byte {
```

Comment:
nitpick: looks like `dbKey` is only used here `[]byte("t")` prefix - can we rename it to something like `blockMetadataInputFeedKey` and take only block number as argument? 
or alternatively we might define a `const blockMetadataInputFeedPrefix = []byte("t")` in the test?